Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM rust:1.57-slim-bullseye as build
+WORKDIR /app
```

Comment:
It is possible to add a user to avoid using `root`? I'm not sure if this has an implication with Cargo though 🤔 

In any case, in other code samples we're using the following:
```
RUN groupadd auth0 && useradd -m developer -g auth0
USER developer
WORKDIR /home/developer/app
```

**PS:** By setting `WORKDIR /home/developer/app` we don't need to add extra lines to give permissions and ownership to the working folder 🙂